store/postgres/src/store.rs

```diff
@@ -818,6 +844,8 @@ impl StoreTrait for Store {
             );
             self.apply_entity_operations_with_conn(&conn, ops, EventSource::None)?;
 
+            self.emit_revert_event(&conn, &subgraph_id, &block_ptr_from, block_ptr_to)?;
```

Comment:
It doesn't really matter: notifications are transactional, so other DB connections won't see the notifications until the current txn commits. I put it there initially because I wasn't sure if the revert proc would also remove the history entries it's rolling back (though I believe that's not the case)

Reply:
Got it, makes sense